code/python/pipelines-workshop/cdk_workshop/cdk_workshop_stack.py

```diff
@@ -3,7 +3,7 @@
     aws_lambda as _lambda,
     aws_apigateway as apigw,
 )
-from cdk_dynamo_table_viewer import TableViewer
+from cdk_dynamo_table_view import TableViewer
```

Comment:
I'm not sure why there are 2 libraries in pip that seem to be the same package by the same author however there are peer dependency differences.

`cdk_dynamo_table_viewer` does not support a recent version of JSII and throws dependency conflict errors with `aws-cdk` packages.

`cdk_dynamo_table_view` is by the same author however it does support a recent/compatible version of JSII.